docs/pages/example/cluster-html.js

```diff
@@ -0,0 +1,10 @@
+/*---
+title: Display HTML clusters with custom properties
+description: An advanced example of using Mapbox GL JS clustering with HTML markers and custom property expressions. To be able to use HTML or SVG for clusters in place of a Mapbox GL layer, we have to manually synchronize the clustered source with a pool of marker objects that we update continuously while the map view changes.
```

Comment:
A few suggestions that (hopefully) simplify the language without making anything untrue: 

>This advanced example uses Mapbox GL JS clustering with HTML markers and custom property expressions. To use HTML or SVG for clusters in place of a Mapbox GL layer, you have to manually synchronize the clustered source with a pool of marker objects that updates continuously while the map view changes.

Reply:
@colleenmcginnis thanks, updated!